app/controllers/admin/people_controller.rb

```diff
@@ -55,12 +55,9 @@ def reorder_role_appointments
 
   def update_order_role_appointments
     current_role_appointment_orderings = @person.current_role_appointments.map(&:ordering)
+    new_ordering = map_ordering_to_current_role_appointment_ordering_values(current_role_appointment_orderings)
```

Comment:
Question: why is this reordering different to all of the other ones in that the values seem to be off-by-one? Might be worth a chat if you have time

Reply:
Yea this is a bit of a snowflake at the moment, and i suspect a follow up PR can probably iron out some of the issues. Basically an ordering has always been passed down explicitly rather than implicitly (based on the order of the collection).

It looks like role appointments are retrieved via link expansion rather than being nested within the person object. Due to this i guess there needs be an explicit way to order their previous roles as you they're ordered on this page 

https://www.gov.uk/government/people/boris-johnson

However, there's an additional need to order a persons current role appointments as the order of a persons roles is used to construct their title.

For example on Rishi Sunaks page https://www.gov.uk/government/people/rishi-sunak

You could shuffle `Minister for the Civil Service` ahead of `First Lord of the Treasury` and the context on that page would move from

`Prime Minister, First Lord of the Treasury, Minister for the Union, and Minister for the Civil Service`

to 

`Prime Minister, Minister for the Civil Service, Minister for the Union, and First Lord of the Treasury`

Historically there has been a need to do this for ministers. I remember it happening a few times since i've joined GDS.

There's probably a way we could order these in Publishing API to ensure the links are ordered in created_at order and then mark specific ones as current to use for reordering the title but that's probably beyond the scope of this. Wdyt?

Reply:
Whatever we decide i think it should be tackled outside of this PR so i'll sort the above comment and merge but happy to pick this up via slack on carry on chatting here 👍